pdns/ws-auth.cc

```diff
@@ -1863,9 +1863,6 @@ static void apiServerZoneRectify(HttpRequest* req, HttpResponse* resp) {
   if (!dk.isSecuredZone(zonename))
```

Comment:
Unrelated to the pull itself, but not allowing rectify for non dnssec zones is also unwanted.
I think we should remove this limitation as well.

Reply:
Maybe this should actually be `if (!dk.isPresigned...`?

Reply:
Yes, rectifying presigned zones will not end well.

Reply:
pushed followup commit for this